pages/_app.js

```diff
@@ -1,14 +1,23 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
+import useLocalStorageState from "use-local-storage-state";
+import Devices from "../db.json";
 
 export default function App({ Component, pageProps }) {
+  const [devices, setDevices] = useLocalStorageState("devices", {
+    defaultValue: [...Devices],
```

Comment:
you don't need to spread the devices here into a new array, and please use a lowercase name, Capital names are reserved for react components and classes.

Reply:
fixed the Capital names, but if I don't spread the "Devices" i run into problems with the unique keys.